build_runner_core/lib/src/generate/performance_tracker.dart

```diff
@@ -10,31 +10,19 @@ import 'dart:async';
 import 'package:build/build.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:meta/meta.dart';
+import 'package:timing/src/timing.dart';
```

Comment:
We should avoid this import, I think the other changes we talked about should allow us to expose publicly everything we need though?